README.md

```diff
@@ -85,7 +85,7 @@ Other Style Guides
 
     bar[0] = 9;
 
-    console.log(foo[0], bar[0]); // => 9, 9
+    console.log(foo[0], bar[0]); // => 1, 9
```

Comment:
in fact, the output will be `9, 9` - that's the entire purpose of the section, to warn about this hazard.